src/DRICH_geo.cpp

```diff
@@ -204,6 +220,11 @@ static Ref_t createDetector(Detector& desc, xml::Handle_t handle, SensitiveDetec
     vesselSolid = vesselBox;
     gasvolSolid = gasvolBox;
     break;
+  case 4:
+  case 5:
+    vesselSolid = vesselBox;
+    gasvolSolid = gasvolBox;
+    break;
   case 2:
     vesselSolid = vesselBox;
     gasvolSolid = gasvolUnion;
```

Comment:
Add `default` case (even though already handled, but code above may change and leave this as the first `switch`).